README.md

```diff
@@ -28,14 +28,17 @@ Andromeda allow:
 
 The idea on making Andromeda possible came by [Jellyfish Insights](http://jellyfishinsights.com/)
 
-## Building and running
-
-The instructions to setup this project can be seen [here](./docs/how_to_setup.md)
-
-## Running via docker container
-
-The instructions to setup this project via docker container can be seen [here](./docs/how_to_run_docker_container.md)
-
-## Contribute to Andromeda
-
-The instruction on how to become a Andromeda contributor can be seen [here](./docs/how_to_contribute.md)
\ No newline at end of file
+# Documentation
+
+* [Getting started](./docs/how_to_setup.md)
+  * [System Requirements](./docs/how_to_setup.md#System-Requirements)
+  * [System Bootstrap](./docs/how_to_setup.md#System-Bootstrap)
+* [Running via Docker container](./docs/how_to_run_docker_container.md)
+  * [Prerequisites](./docs/how_to_run_docker_container.md#Prerequisites)
+  * [Configuring the appsettings.json](./docs/how_to_run_docker_container.md#Configuring-the-appsettings.json)
+  * [Running the container](./docs/how_to_run_docker_container.md#Running-the-container)
+* [Exporting database into files](./docs/export_csv_json.md)
```

Comment:
Maybe:
Exporting data from the data lake into files.

Reply:
Idk, *data* from the *data*.
Exporting data lake into files maybe?

Reply:
> Exporting data lake into files maybe?
This sounds to me as if we were serializing the Database itself.
We are exporting some part of the data from the database. So maybe:
Exporting Data lake data to files.